### PR TITLE
feat(replays): Add replays resources

### PR DESCRIPTION
**Pull request by cmanallen**
https://www.notion.so/sentry/Replay-Backend-Architecture-Update-7-19-7c4ecc8af3fb4027992f3ee75dae9529

### CODE REVIEW

**Review by evanh — COMMENTED** [review]
Looks good from an SnS view.

File: bin/mock-replay
Comment (markstory) [inline bin/mock-replay:67]: You could use click.echo() if you wanted to avoid the noqa.

Comment (cmanallen) [reply bin/mock-replay]: Thanks I updated it.

Also I just pushed minor changes to the test suite.  URL ingestion was merged upstream so I needed to assert that they existed in clickhouse.  Shouldn't affect any reviewing you might have done.

File: src/sentry/replays/post_process.py
Comment (cmanallen) [reply src/sentry/replays/post_process.py]: That's really nice.  I'm saving this because I think the feature will comeback for beta.  For now, the alpha release won't support transactions so I've removed the stubs.

**Comment by cmanallen** [comment]
@markstory Just so you're aware, the tests fails because I'm waiting on open pulls on the snuba repo.  Once those are merged this can be merged too (assuming its approved).

https://github.com/getsentry/snuba/pull/3021
https://github.com/getsentry/snuba/pull/3023

**Comment by cmanallen** [comment]
@markstory Tests passing.  Let me know if you have any more reviews.

**Review by markstory — APPROVED** [review]

**Pull request merged**
